bin/limitd

```diff
@@ -16,4 +43,8 @@ function finish () {
 
 process
   .on('SIGTERM', finish)
-  .on('SIGINT', finish);
+  .on('SIGINT', finish)
+  .on('uncaughtException', (err) => {
+    logger.error(err);
+    finish();
```

Comment:
shouldn't this past the error to finish so the process terminates with a code !== 0?